assets/webpack.config.js

```diff
@@ -143,7 +209,9 @@ module.exports = (env, argv) => {
       },
       plugins: [
         new MiniCssExtractPlugin({ filename: "../css/[name].css" }),
-        new CopyWebpackPlugin({ patterns: [{ from: "static/", to: "../" }] }),
+        new CopyWebpackPlugin({
+          patterns: [{ from: "static/fonts", to: "../fonts" }],
```

Comment:
Remind me why we copy all of `static` in the non-DUP config but only `static/fonts` here?

Reply:
I thought at first it was a way to reduce package size but a quick test proved that wrong. No reason for them to be different.

Reply:
Excellent — then I think some further deduplication is possible with these plugins.